tools/jetty/webapps/portal/index.html

```diff
@@ -1,75 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes. But... have you tested it? i.e. build my branch, `run bin/startup.sh` and then visit `http://localhost:8080`.

I think/hope you will find the redirect and everything still works fine.